File: String_Functions/Frequency_Array/freq.cpp
#include <iostream>
#include <vector>
#include <unordered_map>

using namespace std;

vector<int> freqArr(string arr)
{
    vector<int> dc(26, 0);
    for (char i : arr)
        dc[i - 97]++;
    return dc;
}

int main()
{
    string vc = "codestock";
    vector<int> mp = freqArr(vc);
    for (int i = 0; i < 26; i++)
        cout << (char)(i + 97) << " " << mp[i] << '\n';
    return 0;
}

File: String_Functions/Frequency_Array/freq.py
def freqArr(mystr):
    dc = [0]*26
    for i in mystr:
        dc[ord(i)-97]+=1
    return dc

if __name__ == "__main__":
    mystr = "codestock"
    mp = freqArr(mystr)
    for idx, v in enumerate(mp):
        print(chr(idx+97),v)

